### PR TITLE
BrowserConsole Documentation

### PR DESCRIPTION
**Pull request by bbrennanbasis**
Pull request for a new page called "Console Logging" in "Advanced Topics".

### CODE REVIEW

**Review by MatthewHawkins — APPROVED** [review]

File: docs/advanced_topics/console-logging.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+<!-- :::info
+Before `24.10`, the `App.consoleLog()` and `App.consoleErr()` methods enabled this behavior, but they have since been marked for deprecation.
+::: -->
+
+## Instance
+
+Get an instance of `BrowserConsole` using the `App.console()` method. Print any `Object` desired as one of the five log types: log, info, warn, error, or debug.
+
+```java
```
Comment (hyyan) [inline docs/advanced_topics/console-logging.md:19]: at the top of this code block , show how to import the method staticlly :

```
import static com.webforj.App.console;
```

File: docs/advanced_topics/console-logging.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    .weight().bolder()
+    .size().larger()
+    .color().gray()
+    .style().italic()
+    .transform().uppercase()
+    .background().blue()
+    .warn("Mixing styles");
+```
+
+:::note
```
Comment (hyyan) [inline docs/advanced_topics/console-logging.md:130]: remove this

**Review by hyyan — CHANGES_REQUESTED** [review]

**Comment by hyyan** [comment]
@bbrennanbasis  looks good, thanks. I asked for few minor changes

File: docs/advanced_topics/browser-console.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+title: Browser Console
+---
+
+import ComponentDemo from '@site/src/components/DocsTools/ComponentDemo';
+import JavadocLink from '@site/src/components/DocsTools/JavadocLink';
+
+Using the browser's console to print valuable program information is an integral part of the development process. The <JavadocLink type="foundation" location="com/webforj/BrowserConsole" code='true'>BrowserConsole</JavadocLink> utility class comes with a slew of features to enhance logging capabilities.
+
+<!-- :::info
+Before `24.10`, the `App.consoleLog()` and `App.consoleErr()` methods enabled this behavior, but they have since been marked for deprecation.
```
Comment (hyyan) [inline docs/advanced_topics/browser-console.md:12]: Tha should be **App.consoleError**

**Review by hyyan — CHANGES_REQUESTED** [review]

File: docs/advanced_topics/browser-console.md (hunk 3 above)
Comment (bbrennanbasis) [reply docs/advanced_topics/browser-console.md]: Ok, I've corrected that and corrected it in 24.10.md (The 24.10 blog post under Breaking changes > Console logging).

**Comment by hyyan** [comment]
@bbrennanbasis please also fix the conflict with the `24.10.md`

File: docs/advanced_topics/browser-console.md (hunk 3 above)
Comment (hyyan) [reply docs/advanced_topics/browser-console.md]: @bbrennanbasis Thanks, but this is causing a conflict with the latest content in the `24.10.md` file. You'll need to rebase the current branch onto the `website` branch first.

**Review by hyyan — APPROVED** [review]

**Pull request merged**